src/screens/SearchScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+
+import EditScreenInfo from 'components/EditScreenInfo';
+import { useEffect, useState } from 'react';
+import { getAllTrees } from 'src/database/firebase';
+import { Tree } from '@types';
+import SearchCard from 'src/components/SearchCard';
+import { Searchbar, Title } from 'react-native-paper';
+import ViewContainer from 'src/components/ViewContainer';
+
+export default function SearchScreen() {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [filteredTrees, setFilteredTrees] = useState<Tree[]>([]);
```

Comment:
Since `filteredTrees` is directly dependent on `searchQuery`, I believe we don't need a third state variable here.

It might be better to declare a const. eg. 

```
const results = trees.filter(...)
...
{ results.map((tree: Tree)) => (
   <SearchCard key={tree.id} treeName={tree.name} treeID={tree.id} /> 
)}
```

Reply:
a little confusing at first `onSearchChange` changes `searchQuery` which changes the `results` on re-render. remove `filteredTrees` altogether